### DISCUSSION TITLE
Solution to Using react-native-tab-view breaks when inside of a ScrollView

### DISCUSSION THREAD

**Discussion by ZuhirL** [react-navigation/react-navigation#11352] [General]
Regarding this discussion https://github.com/react-navigation/react-navigation/issues/11067
you can put the scroll view inside the component={HomeScreen} and put it like this.
So you can scroll the page and the scroll bar is invisible

```
const HomeScreen= () => {
  return (
    <ScrollView 
      showsVerticalScrollIndicator={false}
    >
      {/* Your content here */}
    </ScrollView>
  );
};
```

**Comment by okwasniewski** [comment]
The discussion you mentioned was related to embedding the whole TabView inside of ScrollView - not a single route.
